src/app_engine/apprtc.py

```diff
@@ -256,8 +256,6 @@ def get_room_parameters(request, room_id, client_id, is_initiator):
 
   debug = request.get('debug')
   if debug == 'loopback':
-    # Set dtls to false as DTLS does not work for loopback.
-    dtls = 'false'
```

Comment:
Please include something in the PR description about this so one doesn't have to look up #312 and #34